coderd/database/migrations/000205_unique_linked_id.up.sql

```diff
@@ -0,0 +1,18 @@
+-- Remove the linked_id if two user_links share the same value.
+-- This will affect the user if they attempt to change their settings on
+-- the oauth/oidc provider. However, if two users exist with the same
+-- linked_value, there is no way to determine correctly which user should
+-- be updated. Since the linked_id is empty, this value will be linked
+-- by email.
+UPDATE ONLY user_links out
```

Comment:
```suggestion
UPDATE ONLY user_links AS out
```
Not necessary, but perhaps more clear.

PS. Is ONLY actually necessary here, is there inheritance involved?

Reply:
Yes, only because of inheritance.